 08-readline-history.h.diff by Pawel Wiecek <coven@debian.org>
 (compatibility with older versions)

 Setting the global variable rl_inhibit_completion to 1 did indeed
 disable completion, but the invoking key (e.g., <TAB>) was not
 inserted directly into the input line as it should be.
 This patch fixes this problem, and also declares rl_inhibit_completion
 in readline.h for better readline compatibility.

--- a/libedit/readline/Makefile	2003-08-03 19:23:15.000000000 +1000
+++ b/libedit/readline/Makefile	2009-06-23 08:35:58.000000000 +1000
@@ -10,4 +10,6 @@ INCS=		readline.h
 INCSDIR=	/usr/include/readline
 INCSYMLINKS=	readline.h ${INCSDIR}/history.h
 
+incinstall:: linksinstall
+
 .include <bsd.prog.mk>
